srcpkgs/ReText/template

```diff
@@ -1,34 +1,29 @@
 # Template file for 'ReText'
 pkgname=ReText
-version=7.0.4
-revision=2
+version=7.1.0
+revision=1
 archs=noarch
```

Comment:
Please remove `archs=noarch`, we are deprecating it.

Reply:
removed.